frontend/src/pages/modelRegistry/screens/ModelVersionDetails/const.ts

```diff
@@ -8,14 +8,11 @@ export enum ModelVersionDetailsTabTitle {
   DEPLOYMENTS = 'Deployments',
 }
 
-export const pipelineRunSpecificKeys: string[] = [
-  '_registeredFromPipelineProject',
-  '_registeredFromPipelineRunId',
-  '_registeredFromPipelineRunName',
-];
-
-export type PipelineModelCustomProps = {
-  project: string;
-  runId: string;
-  runName: string;
+export type ModelVersionPipelineDescriptionProps = {
+  sourceInfo: {
+    project: string;
+    runId: string;
+    runName: string;
+  };
```

Comment:
Ok, we do repeat `project`, `runId` and `runName` a fair amount. We can reuse the type formerly named `PipelineModelCustomProps`, with the new name `PipelineRunReference` or whatever we want to call it.